script.js

```diff
@@ -37,17 +36,17 @@ const setup = async () => {
   //   });
 };
 
-function compare(a,b) {
-  const nameA = a.name
-  const nameB = b.name
+function compare(a, b) {
+  const nameA = a.name;
+  const nameB = b.name;
 
   let comparison = 0;
-  if(nameA>nameB) {
+  if (nameA > nameB) {
     comparison = 1;
-  } else if (nameA<nameB){
+  } else if (nameA < nameB) {
     comparison = -1;
   }
-  return comparison
+  return comparison;
 }
 
 const searchField = document.getElementById("search-field");
```

Comment:
Think of dynamically updating the `searchField placeholder`  to match the content on the page. Currently it says `Search for episodes` when the shows list is displayed as well.